index.html

```diff
@@ -117,6 +117,25 @@ <h2>Code</h2>
     level whenever inappropriate behaviors are observed.</li>
 </ol>
 </section>
+	  
+<section id="Reporting">
+	<h2>Reporting</h2>
+	<p>Violations of the code of conduct should be reported to a W3C Ombudsperson as outlined in the complementary <a href="https://www.w3.org/Consortium/pwe/#Procedures">Procedures</a> document.</p>
```

Comment:
Suggest changing: 

> Violations of the code of conduct should be reported

to:

> When disagreements within the scope of the code of conduct can not be resolved informally they should be reported

which is in line with the change proposed in #33 . I presume the intention is not to burden everyone working on W3C work to report every disagreement to the Ombudspeople.

Reply:
agreed to this change in 4/4/2019 meeting